Tools/CudaVersionUpdateTool/CudaVersionUpdateTool.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Should we create a parent solution for "all" tools (including future tools)?

Reply:
Makes sense. I have updated the PR to create a consolidated Tools solution.